src/palace/manager/api/circulation.py

```diff
@@ -570,6 +570,33 @@ class BaseCirculationApiSettings(BaseSettings):
         )
     }
 
+    subscription_activation_date: str | None = FormField(
+        default=None,
+        form=ConfigurationFormItem(
+            label=_("Collection Subscription Activation Date"),
+            type=ConfigurationFormItemType.DATE,
+            description=(
+                "A date before which this collection is considered inactive. Associated libraries"
+                " will not be considered to be subscribed until this date). If not specified,"
+                " it will not restrict any associated library's subscription status."
+            ),
+            required=False,
+        ),
+    )
+    subscription_expiration_date: str | None = FormField(
+        default=None,
+        form=ConfigurationFormItem(
+            label=_("Collection Subscription Expiration Date"),
+            type=ConfigurationFormItemType.DATE,
+            description=(
+                "A date after which this collection is considered inactive. Associated libraries"
+                " will not be considered to be subscribed beyond this date). If not specified,"
+                " it will not restrict any associated library's subscription status."
+            ),
+            required=False,
+        ),
+    )
```

Comment:
Should we be adding these settings for every circulation API? It seems like we only really want them for OPDS importer based APIs.

Reply:
I think they could be handy more generally; but, I'm fine moving them to the OPDS collections setting hierarchy for now.

I'm planning to implement the subscription (vs. association) concept across all collection types, but the absence of these settings on a collection would be interpreted as always subscribed (i.e., no restriction).